### ISSUE TITLE
GLES2 `set_mesh()` results in error message "OpenGL ES 2.0 does not allow retrieving blend shape data"

### ISSUE THREAD

**Issue by esbvaGames** [godotengine/godot#48427]
**Godot version:**

Godot 3.3.x rc9

**OS/device including version:**

OPENGL 3.0  - INTEL (R) HD GRAPHICS 620
OPENGL 2.0  - INTEL (R) HD GRAPHICS 620

**Issue description:**

This procedure works correctly, in opengl 2.0, and 3.0, in the editor, but it means a lot of Warning, unnecessary in the DEBUG exported application, although it works correctly, in version 3.1.1, it worked without giving those messages, there should be a way to disable them in the console, there is no checkbok to hide it.

**Steps to reproduce:**

![ConsoleBuggs](https://user-images.githubusercontent.com/5530499/116925659-7285b780-ac27-11eb-95af-dff5c0ca86c5.png)
[ConsoleBugs.zip](https://github.com/godotengine/godot/files/6417313/ConsoleBugs.zip)

extends Area

# To see the messages from the console, export this project, as DEBUG.-
# Key button pressed,  1,2,3
```
func _ready():
	#. Editor mesh Porperties MAKE_UNIQUE
	var newMesh = $Cube.get_mesh().duplicate()
	print('new Mesh ...: ', newMesh)
	
	#. Editor material Properties MAKE_UNIQUE
	var newMaterial = $Cube.mesh.surface_get_material(0).duplicate()
	print('New Material: ', newMaterial)
	
	#. Editor Mesh Local to Scene
	$Cube.set_mesh(newMesh)
	$Cube.mesh.surface_set_material(0, newMaterial)
	pass
```
**Minimal reproduction project:**

**Comment by Calinou** [comment]
> To see the messages from the console, export this project, as DEBUG.-

Well, you won't see those messages in a project exported in release mode :slightly_smiling_face:

**Comment by esbvaGames** [comment]
Precisely, I need all the messages, in the console except this one, because it is not an error, and as a warning it does not work at all, since the procedure works!

**Comment by akien-mga** [comment]
Fixed by #48480.
